### PR TITLE
Use Multi with function when it's needed.

### PR DESCRIPTION
**Pull request by nashby**

### CODE REVIEW

File: lib/hexpm/repository/releases.ex [hunk 1]
```diff
@@ -190,21 +187,19 @@ defmodule Hexpm.Repository.Releases do
...
           %{release | package: package}
-          |> repo.preload(requirements: Release.requirements(release))
+          |> Repo.preload(requirements: Release.requirements(release))
           |> Release.update(params, checksum)
         else
           Release.build(package, params, checksum)
         end
 
       changeset
       |> validate_reserved_version(reserved_packages)
```
Comment (ericmj) [inline lib/hexpm/repository/releases.ex]: Let's remove the pipe here now.

File: lib/hexpm/accounts/audit_log.ex [hunk 2]
```diff
@@ -74,9 +74,9 @@ defmodule Hexpm.Accounts.AuditLog do
   end
 
   def audit_with_user(multi, {_user, user_agent}, action, fun) do
-    # TODO: Use new Multi.insert with functions
-    Multi.merge(multi, fn %{user: user} = data ->
-      Multi.insert(Multi.new(), multi_key(action), build(user, user_agent, action, fun.(data)))
+    multi
+    |> Multi.insert(multi_key(action), fn %{user: user} = data ->
```
Comment (ericmj) [inline lib/hexpm/accounts/audit_log.ex]: We try to avoid single pipes since they don't really add a lot.

**Comment by nashby** [comment]
@ericmj fixed!

**Comment by ericmj** [comment]
Thank you! 💜

**Pull request merged**